.gitignore

```diff
@@ -91,3 +91,4 @@ node_modules/
 
 # Yarn Integrity file
 .yarn-integrity
+/.vs
```

Comment:
Do you use Visual Studio 2022 for this? Get Visual Studio Code :-)

Reply:
yeah I don't know for some reason I feel more comfortable with vs, I will try code as well